src/core/vector.h

```diff
@@ -18,6 +18,13 @@
         x ## _alloc = 0; \
     } while (0)
 
+#define MVM_VECTOR_SIZE(x) \
+    sizeof(*x) * (x ## _alloc)
+
+#define MVM_VECTOR_TOP(x) \
+    (x) + (x ## _num)
```

Comment:
those should use outer parentheses, so `MVM_VECTOR_TOP(x) * 5` works as intended

Reply:
Done, thanks! although `MVM_VECTOR_TOP(x) * 5` makes little sense as that is a pointer :smile: